core/src/main/java/org/apache/druid/data/input/impl/prefetch/JsonIterator.java

```diff
@@ -43,7 +43,7 @@
 {
   private JsonParser jp;
   private ObjectCodec objectCodec;
-  private final TypeReference typeRef;
+  private final TypeReference<T> typeRef;
```

Comment:
Are these changes required for something or just opportunistic?

Reply:
They are required after upgrading Jackson to 2.10